ui/effects/use-play-next.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import React from 'react';
+import { generateListSearchUrlParams, formatLbryUrlForWeb } from 'util/url';
+import { useHistory } from 'react-router';
+
+// Returns web blob from the streaming url
+export default function usePlayNext(
+  isFloating: boolean,
+  collectionId?: string,
```

Comment:
Minor:  I think the signature should be `collectionId: ?string` for this case.  Same goes for the few below ... they aren't optional (must be passed in), but they can be null/undefined.